src/egui_node.rs

```diff
@@ -324,7 +327,9 @@ impl Node for EguiNode {
             index_offset += mesh.vertices.len() as u32;
 
             let texture_handle = match mesh.texture_id {
-                egui::TextureId::Managed(id) => EguiTextureId::Managed(self.window_entity, id),
+                egui::TextureId::Managed(id) => {
+                    EguiTextureId::Managed(self.window_entity_render.id(), id)
```

Comment:
`EguiTextureId::Managed` is actually checked in a main world schedule ; so this should be `window_entity_main` !